### PR TITLE
Node view

### LINKED ISSUE #18: Create nodes panel

### PR DESCRIPTION
**Pull request by nityanandagohain**
image

### CODE REVIEW

File: pkg/gui/layout.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func (gui *Gui) layout(g *gocui.Gui) error {
 		clusterInfoView.Highlight = true
 	}
 
-	namespaceViewHeight := termHeight - unitHeight - 1
+	namespaceViewHeight := termHeight - unitHeight - 7
```
Comment (yolossn) [inline pkg/gui/layout.go]: The height of namespaceView can be reduced and nodeView can be increased by 1 unitHeight

File: pkg/gui/node.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+func (gui *Gui) reRenderNodeInfo() error {
+
+	nodeView := gui.getNodeInfoView()
+	if nodeView == nil {
+		return nil
+	}
+
+	if gui.getNSCount() == 0 {
```
Comment (yolossn) [inline pkg/gui/node.go]: Must be changed to NodeCount here

Comment (GnaneshKunal) [reply pkg/gui/node.go]: Why?

Comment (yolossn) [reply pkg/gui/node.go]: why check Namespace count when rendering node ?

**Comment by yolossn** [comment]
LGTM. Later we have to add handler functions for click on node.

**Pull request merged**